frontend/src/components/PMMainPage.js

```diff
@@ -70,29 +72,32 @@ class PMMainPage extends Component {
 
   render() {
     return (
-      <div style={{ textAlign: 'center' }}>
+      <div className="page">
+        <div>
+          <span />
+          <p>Fieldy McPartnerson</p>
+        </div>
         <h2>Field Partners</h2>
 
         <form onSubmit={this.handleSubmit}>
-          <label>
-            Q:
-            <input
-              type="text"
-              value={this.state.query}
-              placeholder="Search for a Field Partner..."
-              onChange={this.handleQueryChange}
-            />
-          </label>
+          <img src={search} width="18" />
+          <input
+            className="input-master"
+            type="text"
+            value={this.state.query}
+            placeholder="Search for a Field Partner..."
+            onChange={this.handleQueryChange}
+          />
         </form>
 
-        <Tabs>
-          <TabList>
-            <Tab>Reviewing</Tab>
-            <Tab>Dormant</Tab>
+        <Tabs className="tab-master">
+          <TabList className="react-tabs__tab-list">
```

Comment:
is there a reason for the double underscore in the className? This might just be a typo but just wanted to make sure

Reply:
Nah. It's the default className used for Tabs.